articles/cdr-scale-barriers/components/download-report.js

```diff
@@ -0,0 +1,52 @@
+import { Box } from 'theme-ui'
+import { Link, Button } from '@carbonplan/components'
+import { RotatingArrow } from '@carbonplan/icons'
+
+const sx = {
+  height: [24, 24, 34, 46],
+  width: [24, 24, 34, 46],
+  verticalAlign: 'middle',
+  ml: ['4px', '4px', '4px', '6px'],
+  transition: 'color 0.15s, transform 0.15s',
+}
+
+const DownloadReport = () => {
+  return (
+    <Box
+      sx={{
+        borderTop: ({ colors }) => `solid 1px ${colors.muted}`,
+        borderBottom: ({ colors }) => `solid 1px ${colors.muted}`,
+        py: [4],
+        my: [6, 6, 6, 7],
+      }}
+    >
+      <Button
+        size='lg'
+        href='https://files.carbonplan.org/CDR-Scale-Barriers.pdf'
+        suffix={
+          <>
+            <RotatingArrow
+              className='arrow'
+              sx={{ ...sx, ml: [0], color: 'teal' }}
+            />
+            <RotatingArrow className='arrow' sx={{ ...sx, color: 'green' }} />
+            <RotatingArrow className='arrow' sx={{ ...sx, color: 'orange' }} />
+            <RotatingArrow className='arrow' sx={{ ...sx, color: 'yellow' }} />
+          </>
+        }
+        sx={{
+          '&:hover > #suffix-span > .arrow': {
```

Comment:
nit: It seems a little brittle to reference the internal ids defined in `Button`. If we put a class on a span wrapping the `RotatingArrows` we could do `&:hover .arrow-wrapper > .arrow`

Reply:
great idea, fixed!